docs/reference/installation.rst

```diff
@@ -1,20 +1,29 @@
+.. index::
+    single: Installation
+    single: Configuration
+
 Installation
 ============
 
 Prerequisites
 -------------
 
-PHP 7.1 and Symfony 3.4 are needed to make this bundle work ; there are also some
-Sonata dependencies that need to be installed and configured beforehand:
+PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand.
+
+Required dependencies:
+
+* `SonataAdminBundle <https://sonata-project.org/bundles/admin>`_
+* `SonataCacheBundle <https://sonata-project.org/bundles/cache>`_
+* `SonataBlockBundle <https://sonata-project.org/bundles/block>`_
+* `SonataNotificationBundle <https://sonata-project.org/bundles/notification>`_
 
-    - `SonataCacheBundle <https://sonata-project.org/bundles/cache>`_
-    - `SonataBlockBundle <https://sonata-project.org/bundles/block>`_
-    - `SonataEasyExtendsBundle <https://sonata-project.org/bundles/easy-extends>`_
-    - `SonataNotificationBundle <https://sonata-project.org/bundles/notification>`_
-    - `SonataAdminBundle <https://sonata-project.org/bundles/admin>`_
-    - `SonataDoctrineORMAdminBundle <https://sonata-project.org/bundles/doctrine-orm-admin>`_
+And the persistence bundle (in this case not all Sonata persistence bundles are compatible):
```

Comment:
Great!
In order to avoid the idea that there are compatibility issues with other managers, I'd say:
>(currently, not all the implementations of the Sonata persistence bundles are available):

Reply:
Perfect, changing it and applying it to the other bundles where this exact problem happens